docs/user-documentation/user-intro.md

```diff
@@ -35,7 +35,7 @@ Even though Fedora is optional in Islandora, most repositories will use it since
 * flexible, and configurable, disk storage architecture
 * fixity digest generation
 * Memento versioning
-* integration with RDF/Linked Data triplestores
+* integration with RDF/Linked Data triple-stores
```

Comment:
Should be triplestore as it refers to https://en.wikipedia.org/wiki/Triplestore. Maybe a link here would be helpful.